src/element/textElement.test.ts

```diff
@@ -42,9 +40,7 @@ describe("Test wrapText", () => {
       {
         desc: "break all words when width of each word is less than container width",
         width: 80,
-        res: `Hello 
-whats 
-up`,
+        res: `Hello \nwhats \nup`,
```

Comment:
had to replace with single-line strings because the test relied on a trailing whitespace, which we're stripping on file save (or at should be): https://github.com/excalidraw/excalidraw/blob/master/.editorconfig#L12

Reply:
That sounds like voodoo — there's no difference between single-line strings and multi-line strings. Multi-line strings are actually more "dangerous" as this file shows because the trailing whitespaces are invisible unless you print invisible characters (which vscode I believe can't even do).